Practice (#109)
resource-watch/data-pre-processing: Pending_updates/cit_022_road_traffic_death_rates/README.md

## Road Traffic Death Rates Dataset Pre-processing
This file describes the data pre-processing that was done to [the Estimated road traffic death rate (per 100 000 population)](http://apps.who.int/gho/data/node.wrapper.imr?x-id=198) for [display on Resource Watch](https://resourcewatch.org/data/explore/6b670396-c52c-430c-b5bb-20693da03b60).

> The source provided the dataset in a .csv format.

Reviewer: ```suggestion
The source provided the dataset as a csv file.
```

> Below, we describe the steps used to reformat the table so that it is formatted correctly to upload to carto.  
> 

Reviewer: ```suggestion
Below, we describe the steps used to reformat the table so that it is formatted correctly to upload to Carto. 
```

> 1. We read in the data as a pandas dataframe and renamed all the columns.

Reviewer: ```suggestion
1. Read in the data as a pandas dataframe and rename all the columns.
```
